test/regex/allocationCounts.good

```diff
@@ -0,0 +1,18 @@
+Search no capture
+
+Search with capture
+0: file.chpl:line: allocate 8B of string copy data at <address>
+0: file.chpl:line: allocate 8B of string copy data at <address>
+0: file.chpl:line: free at <address>
+0: file.chpl:line: allocate 8B of string copy data at <address>
+0: file.chpl:line: allocate 8B of string copy data at <address>
+0: file.chpl:line: free at <address>
+0: file.chpl:line: allocate 8B of string copy data at <address>
+0: file.chpl:line: allocate 8B of string copy data at <address>
+0: file.chpl:line: free at <address>
+(aaa, bbbbb, ccccc)
+
+Iter Matches no capture
+
+Iter Matches with capture
+
```

Comment:
Any reason to skip having a test for iter split?

Reply:
Only because we don't expect to see anything interesting